src/main/scala/org/broadinstitute/dsde/snoop/ws/SnoopApiFormats.scala

```diff
@@ -1,34 +1,23 @@
 package org.broadinstitute.dsde.snoop.ws
 
-import spray.json.DefaultJsonProtocol
+import org.broadinstitute.dsde.snoop.ws.WorkflowParameter.WorkflowParameter
+import spray.json._
 
-class SnoopApiFormats {
-  
+object WorkflowParameter {
+  type WorkflowParameter = Either[String, Seq[String]]
```

Comment:
I think this is a good use of Either, as long as it's always a binary union, which I can't imagine that changing. If it does change we can go more along the route of the other thing I mentioned or a typeclass.

Related: this was a decent article on union types in scala http://www.chuusai.com/2011/06/09/scala-union-types-curry-howard/